.github/workflows/integration-test.yaml

```diff
@@ -98,7 +121,8 @@ jobs:
           TEST_PROJECT_CODE: 'sena-3'
           TEST_DEFAULT_PASSWORD: ${{ secrets.TEST_USER_PASSWORD }}
         working-directory: frontend
-        run: pnpm test
+        # Timeout of 1 minute instead of 30s, so slow runners will still have a chance to complete their work
+        run: pnpm test -- --timeout=60000
```

Comment:
I don't think we need a 1 minute timeout anymore, the tests are running much faster, I think if we have to set the global timeout to 1 minute then we need to solve that some other way, that's just too long to wait.

Reply:
Done in commit 9c3718eb.